conformance/privileged-mcp-action-v0/scripts/score_candidate.py

```
@@ -0,0 +1,373 @@
+        raise CandidateError("invalid report must omit claims")
+    return value
+
+
+def run_candidate(
+    command: list[str],
+    bundle: Path,
+    timeout_seconds: int,
+) -> dict[str, Any]:
+    with tempfile.TemporaryFile() as stdout, tempfile.TemporaryFile() as stderr:
+        try:
+            completed = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)